pom.xml

```diff
@@ -75,6 +93,10 @@
     </pluginRepositories>
 
     <properties>
-        <openmrsPlatformVersion>2.0.6</openmrsPlatformVersion>
+        <openmrsPlatformVersion>2.2.0</openmrsPlatformVersion>
```

Comment:
Why are we changing the `openmrsPlatformVersion`?

Reply:
My  intension was to update this version, but again it has not issue, i have reverted back